app/helpers/filters_helper.rb

```diff
@@ -56,6 +56,17 @@ def scheme_status_filters
     }.freeze
   end
 
+  def location_status_filters
```

Comment:
I've not seen this before, is this shorthand for a switch statement? 
If it's doing what I think it is then I would assume there's a helper function that takes a string, removes replaces the underscores with spaces and capitalises the first letter?

Reply:
Do you mean the hash? https://ruby-doc.org/core-2.7.0/Hash.html
We're just accessing values using keys, similarly like you'd access values in arrays using indexes